nion/swift/ImageCanvasItem.py

```diff
@@ -1157,6 +1257,7 @@ def mouse_position_changed(self, x: int, y: int, modifiers: UserInterface.Keyboa
         image_position = widget_mapping.map_point_widget_to_image(mouse_pos)
         if delegate.image_mouse_position_changed(image_position, modifiers):
             return True
+
```

Comment:
Is there a reason to add a blank line here?

Reply:
Preference for white space, but other than that no. Removed.